### PR TITLE
Add missing custom-* data (part 6)

### PR DESCRIPTION
**Pull request by Carlgo11**
Also:
* Deleted saaspass as when reading through their documentation I got the impression that they're more of a 2FA __provider__.

### CODE REVIEW

File: entries/s/sailpoint.com.json [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
       "custom-software",
       "custom-hardware"
     ],
+    "custom-hardware": [
+      "RSA SecurID",
+      "Symantec VIP"
+    ],
+    "custom-software": [
+      "Duo"
+    ],
```
Comment (Carlgo11) [inline entries/s/sailpoint.com.json:17]: Source: https://github.com/2factorauth/twofactorauth/pull/3451

File: entries/r/rb.cz.json [hunk 2]
```diff
@@ -1,11 +1,12 @@
 {
   "Raiffeisen Bank [CZ]": {
     "domain": "rb.cz",
-    "url": "https://www.rb.cz/",
-    "tfa": [
-      "custom-hardware"
-    ],
-    "documentation": "https://www.rb.cz/firemni-finance/podnikatele-a-male-firmy/prime-bankovnictvi/sluzby-pro-firemni-ucty/bezpecnost-internetoveho-bankovnictvi/",
```
Comment (Carlgo11) [inline entries/r/rb.cz.json:8]: Their documentation states that this is only used to confirm certain actions, and not for accessing the account.

File: entries/p/payeer.com.json [hunk 3]
```diff
@@ -6,8 +6,10 @@
       "email",
       "custom-software"
     ],
-    "documentation": "https://payeer.com/en/faq/",
-    "notes": "Receiving codes via SMS cost 0.05$ each message. 2FA is personalizable in account settings.",
+    "custom-software": [
+      "Telegram"
+    ],
+    "notes": "2FA via SMS costs ¢25 USD per message.",
```
Comment (Carlgo11) [inline entries/p/payeer.com.json]: Created an account and verified this myself.
(I did however forget to take a screenshot before throwing away the account :see_no_evil: )

File: entries/p/passpack.com.json [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
-      "email",
+      "totp",
       "custom-hardware"
     ],
-    "documentation": "https://help.passpack.com/knowledgebase/idx.php/44/0/",
+    "custom-hardware": [
+      "Yubico OTP"
+    ],
+    "documentation": "https://passpack.com/support/getting-started/two-factor-authentication/",
+    "recovery": "https://passpack.com/support/faq/",
```
Comment (Carlgo11) [inline entries/p/passpack.com.json:12]: Screenshots in the doc shows that Google Auth __or__ Yubikeys can be enabled for 2FA. Thus no U2F.

File: entries/o/otpbanka.hr.json [hunk 5]
```diff
@@ -1,11 +1,12 @@
...
   "OTP banka": {
     "domain": "otpbanka.hr",
-    "url": "https://www.otpbanka.hr/",
     "tfa": [
-      "custom-software",
       "custom-hardware"
     ],
+    "custom-hardware": [
+      "OTP token device"
+    ],
```
Comment (Carlgo11) [inline entries/o/otpbanka.hr.json:9]: The device is described by themselves as an "OTP Token" so I went with their name for it.
It's a normal card-reading device that creates HOTP-codes using the bank card's chip and pin.

File: entries/m/mojedatovaschranka.cz.json [hunk 6]
```diff
@@ -1,11 +1,13 @@
...
   "Datové schránky": {
     "domain": "mojedatovaschranka.cz",
-    "url": "https://www.mojedatovaschranka.cz",
     "tfa": [
       "sms",
       "custom-software"
     ],
+    "custom-software": [
+      "eGovernment Mobile Key"
+    ],
```
Comment (Carlgo11) [inline entries/m/mojedatovaschranka.cz.json:10]: Best English translation I could find :shrug:

File: entries/p/payeer.com.json (hunk 3 above)
Comment (hkamran80) [inline entries/p/payeer.com.json]:
```suggestion
    "notes": "2FA via SMS costs 25¢ USD per message.",
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**